### PR TITLE
[WIP] Update Ember to version 3.8.0

### PR DESCRIPTION
**Pull request by galfert**
At the moment there is still a deprecation warning about ember-browserify left. There is no update for that library yet, and the maintainer recommends to switch to ember-auto-import instead. So far I couldn't get that to work with importing kredits-contracts though.

### CODE REVIEW

File: ether_addresses.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+github id: 843
+ether address: 0xD4a64570B12dA659Ee4BBd41c3509B7b1F9c51AC (Metamask)
+
+               0x0020e20e6C9ACc0c9fE78Fcb6a427Ad0cC8aca18 (Parity)
+
+               0xD4a64570B12dA659Ee4BBd41c3509B7b1F9c51AC (kosmos parity)
```
Comment (skddc) [inline ether_addresses.txt]: Is this a personal notes file that was accidentally committed?

Comment (galfert) [reply ether_addresses.txt]: Damn, yes. This wasn't supposed to be committed.

File: app/components/chart-contributions-by-type/component.js [hunk 2]
```diff
@@ -12,9 +12,20 @@ let categoryColors = {
 export default Component.extend({
 
   contributions: null,
+  chartOptions: null,
+
+  init () {
+    this._super(...arguments);
+
+    this.set('chartOptions', {
```
[the remark this answers is not in the document]
Comment (galfert) [reply app/components/chart-contributions-by-type/component.js]: So the `chartOptions` are not supposed to be changed at any point? Then we can use `Object.freeze()` instead.

[the remark this answers is not in the document]
Comment (galfert) [reply app/components/chart-contributions-by-type/component.js]: Changed it in the new PR (#89).

**Comment by galfert** [comment]
Closing in favour of #89, because of an accidentally committed file.

**Pull request closed without merging**